apps/admin/backend/src/app.ts

```diff
@@ -85,6 +85,7 @@ import {
 } from './cast_vote_records';
 import { generateBallotCountReportCsv } from './exports/csv_ballot_count_report';
 import { adjudicateWriteIn } from './adjudication';
+import { convertFrontendFilter as convertFrontendFileUtil } from './util/filters';
```

Comment:
typo - was this supposed to be `convertFrontendFilterUtil`?

Reply:
Yup! Just fixed, thank you.